playback/src/audio_backend/gstreamer.rs

```diff
@@ -46,7 +46,7 @@ impl Open for GstreamerSink {
         let gst_caps = gst_info.to_caps().expect("Failed to create GStreamer caps");
 
         let sample_size = format.size();
-        let gst_bytes = NUM_CHANNELS as usize * 1024 * sample_size;
+        let gst_bytes = NUM_CHANNELS as usize * 2048 * sample_size;
```

Comment:
Would it make sense to use `set_max_time`? It would require no math on our part and not dependent on format, channel count and whatnot (as far as we're concerned anyway).

Reply:
I don't know that interface, but if that takes away this calculation then I'd be all for it. These lines have always seemed a bit magical to me.

Reply:
Even with `set_max_time` it's still magical but at least we're not doing math. I did a quick look at the docs and they don't mention what the default is if there is one or offer any guidance that I saw?

Reply:
In the new `new-api` branch it might make sense to set this the same as the decoder's buffer size maybe? I don't know. I'd really like to make this less magical in some way.

Reply:
I've tried such things before, and scoured the docs too, but to no avail. If I remember correctly, setting that to the buffer size will lead to a broken pipeline.